<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaMenuPrincipal</class>
 <widget class="QMainWindow" name="TelaMenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1194</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pet Shop Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_logoff">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Fazer Logoff</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto_inicial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>110</height>
     </size>
    </property>
    <property name="text">
     <string>FOTO 3X4</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nome">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>103</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cargo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_criar_usuario">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar Funcionarios/Animais</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>310</width>
      <height>173</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_criar_animal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Animal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
